code/modules/admin/verbs/possess.dm

```diff
@@ -17,7 +17,7 @@
 
 /proc/release(obj/O)
 	set name = "Release Obj"
-	set category = "Object"
+	set category = "IC"
```

Comment:
No, keep it in object. This isn't what you think it is.

Reply:
I know it's not related to IC, you basically take control of an object and can move it around. This is opposed of posses. I'm just axing Object Tab since it would only have 2 verbs left.

Reply:
Object tab is literally every object with verbs nearby